easybuild/easyconfigs/p/PyAEDT/PyAEDT-0.9.9-gfbf-2023b.eb

```diff
@@ -0,0 +1,78 @@
+easyblock = 'PythonBundle'
+
+name = 'PyAEDT'
+version = '0.9.9'
+
+homepage = 'https://aedt.docs.pyansys.com/version/stable'
+description = """PyAEDT is a Python library that interacts directly with the
+Ansys Electronics Desktop (AEDT) API, enabling straightforward and efficient
+automation in your workflow."""
+
+toolchain = {'name': 'gfbf', 'version': '2023b'}
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('hatchling', '1.18.0'),
+]
+
+dependencies = [
+    ('dotNET-Core', '8.0', '', SYSTEM),
+    ('libspatialindex', '1.9.3'),
+    ('Python', '3.11.5'),
+    ('Python-bundle-PyPI', '2023.10'),
+    ('Pillow', '10.2.0'),
+    ('pydantic', '2.7.4'),
```

Comment:
is this really a runtime dep?

Reply:
It seems required by one dependency. Otherwise it failed in sanity check: `pyedb 0.21.0 requires pydantic, which is not installed.`

Reply:
It is used here, cannot be simply removed: https://github.com/ansys/pyedb/blob/5c6f122c88a50fa733bb59f63f6a1c419eeec408/src/pyedb/dotnet/edb_core/materials.py

Reply:
Older pydantic version (2.6.4) works in CI